<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_contrasena">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>403</y>
     <width>391</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: rgb(246, 245, 244);
    border: 1px solid #dcdfe6;
    border-radius: 6px;
    padding: 6px 12px;
    font-size: 20px;
    color: #606266;
    min-height: 18px;
}
</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_imagen_registro">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-7</y>
     <width>451</width>
     <height>721</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>recursos/recursos_usuario/image(4).png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_estatico_identificate">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>623</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: white;
	color:#1061C4;
	border: 2px solid #1061C4;
	border-radius: 15px;
	font-weight: bold;
	font-size: 15px;
}
QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
   </property>
   <property name="text">
    <string>Identifícate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_estatico_yatienescuenta">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>633</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>¿Ya tienes una cuenta?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_estatico_telefono">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>274</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Teléfono:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_estatico_registrotxt">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>163</y>
     <width>401</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Regístrate para comprar tus boletos y gestionar tus reservaciones de forma rápida.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_estatico_registro">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>103</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Registro</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_estatico_contrasena">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>363</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Contraseña:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_telefono">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>313</y>
     <width>391</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    
	background-color: rgb(246, 245, 244);
    border: 1px solid #dcdfe6;
    border-radius: 6px;
    padding: 6px 12px;
    font-size: 20px;
    color: #606266;
    min-height: 18px;
}

</string>
   </property>
   <property name="inputMask">
    <string>9999999999</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_continuar">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>490</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 15px;
	font-weight: bold;
	font-size: 22px;
}
QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
   </property>
   <property name="text">
    <string>Continuar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
